ansible/roles/k3s/tasks/runtime.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Check if k3s agents are already connected
+  service_facts:
+  register: services_state
+
+- name: Initialise and authenticate k3s server and agents
+  vars:
+    k3s_server_name: "{{ hostvars[groups['k3s_server'].0].ansible_host }}"
+    access_ip: "{{ ansible_host }}"
+    services_states: > # getting list of all unique agent service states
```

Comment:
Similar comments on namoing conventions again, and make it clear its agent, so maybe `_k3s_agent_states`?